parliament/parliament.js

```diff
@@ -370,6 +548,158 @@ function findIssue (clusterId, issueType, node) {
   }
 }
 
+// Initializes the parliament with ids for each group and cluster
+// Upgrades the parliament if necessary
+async function initializeParliament () {
+  ArkimeUtil.checkArkimeSchemaVersion(User.getClient(), getConfig('parliament', 'usersPrefix'), MIN_DB_VERSION);
+  Notifier.initialize({
+    issueTypes,
+    debug: ArkimeConfig.debug,
+    prefix: getConfig('parliament', 'usersPrefix'),
+    esclient: User.getClient()
+  });
+
+  Parliament.initialize({
+    debug: ArkimeConfig.debug,
+    esclient: User.getClient(),
+    name: internals.parliamentName,
+    prefix: getConfig('parliament', 'usersPrefix')
+  });
+
+  // fetch parliament file if it exists
+  parliamentFile = require(`${getConfig('parliament', 'file')}`);
+
+  // if there's a parliament file, check that it is the correct version
+  if (parliamentFile && (parliamentFile.version === undefined || parliamentFile.version < MIN_PARLIAMENT_VERSION)) {
+    console.log( // notify of upgrade
+      `WARNING - Current parliament version (${parliamentFile.version ?? 1}) is less then required version (${MIN_PARLIAMENT_VERSION})
+        Upgrading your Parliament...\n`
+    );
+
+    // do the upgrade
+    const upgraded = await upgrade.upgrade(parliamentFile, ArkimeConfig, internals.parliamentName, issues, Parliament);
+    parliamentFile = upgraded.parliament;
+    issues = upgraded.issues;
+
+    try { // write the upgraded files
+      if (Buffer.from(JSON.stringify(parliamentFile, null, 2)).length > 100) {
+        fs.writeFileSync(getConfig('parliament', 'file'), JSON.stringify(parliamentFile, null, 2), 'utf8');
+      }
+      if (!validateIssues()) {
+        fs.writeFileSync(app.get('issuesfile'), JSON.stringify(issues, null, 2), 'utf8');
+      }
+    } catch (e) { // notify of error saving upgraded parliament and exit
+      console.log('Error upgrading Parliament:\n\n', ArkimeUtil.sanitizeStr(e.stack));
+      if (ArkimeConfig.debug) {
+        console.log(parliamentReadError);
+      }
+      throw new Error(e);
+    }
+
+    // notify of upgrade success
+    console.log(`SUCCESS - Parliament upgraded to version ${MIN_PARLIAMENT_VERSION}`);
+  }
+
+  // create parliament in db if it doesn't exist
+  // (there was no parliament file file to upgrade from)
+  try {
```

Comment:
put this above `parliamentFile = ` and don't load parliamentFile or check for upgrade if it exists in db